src/ProjectSystem/Impl/Utilities/PathExtensions.cs

```diff
@@ -0,0 +1,30 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See LICENSE in the project root for license information.
+
+using System;
+using System.IO;
+using System.Text;
+using Microsoft.VisualStudio.ProjectSystem.FileSystemMirroring.Interop;
+using Microsoft.VisualStudio.ProjectSystem.Utilities;
+
+namespace Microsoft.VisualStudio.ProjectSystem.FileSystemMirroring.Utilities {
+    public static class PathExtensions {
+        public static string ToLongPath(this string path) {
```

Comment:
I think we can make this part of `IFileSystem` interface (since it's not just an extension method and in fact requires access to file system)